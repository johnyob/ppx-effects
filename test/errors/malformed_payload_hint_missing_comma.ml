let () =
  try f () with
  | [%effect? E (* missing [,] *) k] -> ()
;;
