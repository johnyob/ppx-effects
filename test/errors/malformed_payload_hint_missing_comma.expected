File "malformed_payload_hint_missing_comma.ml", line 3, characters 4-36:
Error: ppx_effects: invalid [%effect? ...] payload. Expected a pattern for an (effect, continuation) pair.
       Hint: did you mean `[%effect? E, k]`?
